subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -463,6 +469,12 @@ u8_t ll_start_enc_req_send(u16_t handle, u8_t error_code,
 			return BT_HCI_ERR_CMD_DISALLOWED;
 		}
 
+		conn->llcp_req++;
```

Comment:
There is some many duplicates of this logic bellow, can't we make a helper function to check llcp_req against llcp_ack?

Reply:
Agree. Can we agree to create an issue and assign it to me? This is mentioned by @carlescufi to evaluate the use of atomic_CAS instead.

Reply:
No problem, lets have it separately with use of atomic as well.

Reply:
> Agree. Can we agree to create an issue and assign it to me? This is mentioned by @carlescufi to evaluate the use of atomic_CAS instead.

Note: Oticon's mcu does not have atomic_cas.
Otherwise, fully agree about helper function or macro.